### ISSUE TITLE
Using the samples and getitem return an error for some dataset

### ISSUE THREAD

**Issue by thibo73800** [Visual-Behavior/aloception#49]
For some dataset (not all), using the sample return an error when caling `getitem`

To reproduce
```
import alodataset

# Coco Detection Dataset (Works) 
dataset = alodataset.CocoDetectionDataset(sample=True)
frame = dataset.getitem(0)
frame.get_view().render()

# CrowdHumanDataset: Do not work
dataset = alodataset.CrowdHumanDataset(sample=True)
frame = dataset.getitem(0)
frame.get_view().render()
```

Note that this is not only the case for CrowdHumanDataset but for some other datasets as well.

**Comment by thibo73800** [comment]
Impossible to download SintelFlow sample.

**Comment by thibo73800** [comment]
Overall, testing the datasetes, I think we have to go though all the datasets to check if everything behave properly with the samples. I get a lot of error getting the items of various datasets. (it might work using strem_loader)

**Comment by Johansmm** [comment]
All these problems were solved in #1
